README.md

```diff
@@ -58,12 +58,20 @@ Some of the services included so far:
 
 ## Getting started
 
+<a href="https://repology.org/metapackage/polybar">
```

Comment:
Markdown has tags for this, e.g:

~~~ md
[![Packaging status](https://repology.org/badge/vertical-allrepos/polybar.svg)](https://repology.org/metapackage/polybar)
~~~